omnibus/cookbooks/firezone/attributes/default.rb

```diff
@@ -381,17 +381,43 @@
 default['firezone']['robots_disallow'] = nil
 
 # ### Outbound Email Settings
-#
 # If none of these are set, the :sendmail delivery method will be used. Using
 # the sendmail delivery method requires that a working mail transfer agent
 # (usually set up with a relay host) be configured on this machine.
-#
-# SMTP will use the 'plain' authentication method.
-default['firezone']['from_email'] = nil
-default['firezone']['smtp_address'] = nil
-default['firezone']['smtp_password'] = nil
-default['firezone']['smtp_port'] = nil
-default['firezone']['smtp_user_name'] = nil
+default['firezone']['outbound_email']['from'] = nil
```

Comment:
Maybe add a comment above this noting that a value of `nil` disables outbound emails? I imagine some paranoid users might want to make sure it's off.

Reply:
Yup. Good point.